stdlib/public/Glibc/CMakeLists.txt

```diff
@@ -1,18 +1,25 @@
 set(sources
-  module.map
+  module.map.in
   )
 set(output_dir "${SWIFTLIB_DIR}/glibc")
 
-set(commands
-    COMMAND ${CMAKE_COMMAND} "-E" "make_directory" "${output_dir}")
-set(outputs)
+# Set correct paths to glibc headers
+set(GLIBC_INCLUDE_PATH "/usr/include")
+set(arch_dir "x86_64-linux-gnu")
+if (EXISTS "${GLIBC_INCLUDE_PATH}/${arch_dir}/sys")
+    set(GLIBC_ARCH_INCLUDE_PATH "${GLIBC_INCLUDE_PATH}/${arch_dir}")
```

Comment:
@gribozavr: How about adding an uncrustify config to the project that has your code formatting so that we run it as a git-hook before making pull requests?